src/content/components/Tooltip.css

```diff
@@ -0,0 +1,69 @@
+.tooltipMount {
+  /* Create a stacking context above the root container element. */
+  position: absolute;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  pointer-events: none;
+}
+
+.tooltip {
+  position: relative;
+  max-width: 100%;
+  padding: 2px 5px;
+  display: inline-block;
+  border: 1px solid #ccc;
+  background-color: #f9f9f9;
+  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.3);
+  text-align: left;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  transition: opacity 250ms;
+  box-sizing: border-box;
```

Comment:
nit: please put `box-sizing` closer to `max-width` as they're related.

bonus points for a new line after `display`